src/main/java/spoon/support/StandardEnvironment.java

```diff
@@ -458,15 +460,17 @@ public void setPreserveLineNumbers(boolean preserveLineNumbers) {
 		this.preserveLineNumbers = preserveLineNumbers;
 	}
 
-	private boolean noclasspath = false;
-
 	@Override
 	public void setNoClasspath(boolean option) {
 		noclasspath = option;
 	}
 
 	@Override
 	public boolean getNoClasspath() {
+		if (this.noclasspath == null) {
+			logger.warn("Spoon is currently use with the default noClasspath option set as true. Read the documentation for more information: http://spoon.gforge.inria.fr/launcher.html#about-the-classpath");
```

Comment:
Do we want to print this warning
A) only once
B) for each call ... to fill the log by these warnings?

I vote for A) ... then add `this.noclasspath = true`

Reply:
I propose to move this warning from `getNoClasspath` to a more higher level method such as `Launcher#run`

Reply:
I'm ok with @pvojtechovsky but I don't understand the purpose of what you propose @monperrus .